dwl.c

```diff
@@ -1398,13 +1398,15 @@ mapnotify(struct wl_listener *listener, void *data)
 void
 monocle(Monitor *m)
 {
-	Client *c;
+	Client *c, *sel;
 
 	wl_list_for_each(c, &clients, link) {
 		if (!VISIBLEON(c, m) || c->isfloating || c->isfullscreen)
 			continue;
 		resize(c, m->w.x, m->w.y, m->w.width, m->w.height, 0);
 	}
+	sel = selclient();
+	wlr_scene_node_raise_to_top(sel->scene);
```

Comment:
```suggestion
	focusclient(focustop(m), 1);
```

Reply:
Right, I'm testing this now and it seems to be the better solution.